src/Microsoft.Azure.WebJobs.Host/Executors/JobHostConfigurationExtensions.cs

```diff
@@ -103,7 +90,9 @@ public static ServiceProviderWrapper CreateStaticServices(this JobHostConfigurat
             services.AddService<TraceWriter>(trace);
 
             // Add built-in extensions 
-            config.AddAttributesFromAssembly(typeof(TableAttribute).Assembly);
+            var metadataProvider = new JobHostMetadataProvider();
```

Comment:
Can these 2 metadata provider lines be moved down below where the metadata provider service is added? Or do they need to be here?